### ISSUE TITLE
Android notifications not working

### ISSUE THREAD

**Issue by ZedSphere** [balzack/databag#57]
When receiving messages on an Android device (Tested on a Google Pixel 7 Pro) no notification sounds play regardless of "Enable Notification Sounds" being toggled to the "ON" position within the app and app permissions being granted.

**Comment by balzack** [comment]
There are two places within the app to enable notifications.
1) within the profile screen switch 'enable notifications' to on
2) within the admin dashboard switch 'enable push notifications' to on

Notifications also must be enabled for the app in the OS, but sounds like you already confirmed this.

Notifications only occur when a contact creates a new topic with you, adds a message to a shared topic, or calls you. Notifications won't occur if you just add a message with your account from the browser.

If all that is confirmed, I've seen some users have issues when enabling the tor network. Would this be the case for you?

**Comment by balzack** [comment]
hmmm, it should be working. Are you using anything like fdroid or lineageOS?

Can you create an account with the demo server databag.coredb.org, ad see if that server is able to push notifications to you?

**Comment by fr2875** [comment]
yea lineageOS and f-droid...

**Comment by fr2875** [comment]
I've never had a notification problem with lineageOS
and I've tried with your server with two different accounts and still no notification even when I test the audio calls.

**Comment by balzack** [comment]
I also use lineageOS and notifications with ntfy/UnifiedPush works for me. 

There have been reports that notifications don't work with the tor network enabled. I'm not yet sure why; would this be the case for you?

**Comment by fr2875** [comment]
nop i use 4g. maybe there's a problem with the battery management?

**Comment by fr2875** [comment]
normally nothing needs to be changed for it to work ?

**Comment by balzack** [comment]
Are you using UnifiedPush for notifications?

Google's firebase messaging is not open source, which is required by f-droid. To support push notifications the f-droid build has the integration replaced with UnifiedPush. Let me know if there is an f-droid app that supports push notifications in another way, so I can support that as well.

If you install the google store version of Databag, it should work without UnifiedPush.

**Comment by fr2875** [comment]
I don't have the playstore 
where can I find this official version?

**Comment by fr2875** [comment]
ok so version 1.1 of F-droid works with notification but you need to install ntfy and then databag not the other way round... 
is it normal that it's version 1.5 on the play store and 1.1 on f-droid ?

**Comment by balzack** [comment]
- need to install ntfy and then databag
yes, thanks for calling this out. I need to be more clear about that

- version 1.5 on the play store and 1.1 on f-droid
yes, that is the current (confusing) versioning. The versioning diverged when I released on fdroid, and I need to fix this.

- where can I find this official version
I see you resolved this with ntfy, but let me know if you still want the google services version and I can publish the '.apk' as part of the github release.

Thanks for staying with this project through the difficulties! It helps me learn where I need to improve the databag service and my communication.

**Comment by fr2875** [comment]
yes putting the apk version on github is really useful when you don't have the playstore.

thank you for creating this free, self-hosted alternative messaging service and keeping it alive.

**Comment by balzack** [comment]
I added the apk to the latest release: https://github.com/balzack/databag/releases/download/v1.1.11/databag.apk

That one has been manually built. I'll link it to the main readme if I figure out how to build it with a github action.

Thanks for all the feedback!
